blog/use-google-optimize-in-react-web-for-a-b-testing.md

```diff
@@ -0,0 +1,136 @@
+# Use Google Optimize in React web for A/B Testing
+
+I'll explain how to create an experiment in the Google Optimize (GO) panel to get a simple boolean flag in our JavaScript code, specifically using Reactjs, to display the original version of our web, or the experiment.
+
+I won't focus on how to create experiments or specific configurations. I'll center this post on how to connect GO with JavaScript code.
+
+## Introduction
+
+I needed to integrate Google Optimize to do an [A/B testing](https://en.wikipedia.org/wiki/A/B_testing) in a project I'm working on. I found that GO was made for end users so that they can make little changes (text, CSS styles, etc.) in the website without needing a developer.
```

Comment:
```suggestion
I needed to integrate Google Optimize to do an [A/B testing](https://en.wikipedia.org/wiki/A/B_testing) in a project I'm working on. I've found that GO was made for end users so that they can make little changes (text, CSS styles, etc.) in the website without needing a developer.
```